### DISCUSSION TITLE
Weekly Governance Updates (10 May, 2024)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#1005] [General]
This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

- FIP0065 - Ignore built-in market locked balance in circulating supply calculation [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0065.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/719)
- FIP0067 - PoRep Security Policy & Replacement Sealing Enforcement [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0067.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/727)
- FIP0079 - Add BLS Aggregate Signatures to FVM [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0079.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/840)
- FIP0082 - Add support for aggregated replica update proofs [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0082.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/752)
- FIP0084 - Remove Storage Miner Actor Method `ProveCommitSectors` [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0084.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/899)
- FIP0085: Convert f090 Mining Reserve Actor to Keyless Account Actor [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0085.md) [(Discussion)](https://github.com/filecoin-project/FIPs/discussions/901)
- FIP0090: Non-Interactive PoRep [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0090.md) [(Discussion)](https://github.com/filecoin-project/FIPs/discussions/854)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*

* Update FIP0001 [(PR)](https://github.com/filecoin-project/FIPs/pull/850)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/799)
* On-chain Retrieval Expectations [(PR)](https://github.com/filecoin-project/FIPs/pull/862)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/861) 
* Introducing SealerID [(PR)](https://github.com/filecoin-project/FIPs/pull/993)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/890)

**Last Call** - *Final opportunity for community feedback!* 

NA

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas model adjustment for user programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
- FIP0077 - Add Cost Opportunity For New Miner Creation [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0077.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/780)
- FIP0078 - Remove Restrictions on the Minting of Datacap [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0078.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/844)
- FIP0080 - Phasing Out Fil+ and Restoring Deal Quality Multiplier to 1x [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0080.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/774)
- FIP0081 - Introduce lower bound for sector initial pledge [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0081.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/847)
- FIP0086 - Fast Finality in Filecoin (F3) [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/809)
- FIP0091 - Add Support for Legacy Homestead Ethereum Transactions [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0091.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/962)

 **:fire: HOT :fire: Discussion Topics**

- There is a proposal to enable fast retrieval on Filecoin by creating a second type of proof dubbed "Fast PoRep". Read more [here](https://github.com/filecoin-project/FIPs/discussions/988)
- Filecoin Foundation has recently released the FIP draft for a proposed update to FIP0001. This new proposal contains significant improvements to the existing governance models of Filecoin. Read the full draft [here](https://github.com/filecoin-project/FIPs/pull/850) 
- If you haven't done so, you can register for our monthly Community Governance Calls every Last Monday of the Month. You can register [HERE](https://calendar.google.com/calendar/embed?src=c_909343f97c15e8f23dda6e2612e62fcdee14bceabd8869abe4a52d793bf42b98%40group.calendar.google.com&ctz=America%2FToronto)

---
**Other Governance Updates**

| Milestone         | Date       |
|-------------------|------------|
| Code Freeze       | 23 May 2024|
| Calib Upgrade     | 6 June 2024|
| Mainnet Upgrade   | 27 June 2024|

**FIPs in Scope for NV23 Upgrade**

| FIP Number | Title                                             | Status          |
|------------|---------------------------------------------------|-----------------|
| FIP0065        | [Ignore Built-In Market Locked Balance in Circulating Supply](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0065.md)                                               | Accepted            |
| FIP0079        | [Add BLS Aggregate Signatures to FVM](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0079.md)                                               | Accepted            |
| FIP0082        | [Add Support for Aggregated Replica Update Proofs](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0082.md)                                               | Accepted            |
| FIP0084        | [Remove Storage Miner Actor Method ProveCommitSectors](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0084.md)                                               | Accepted            |
| FIP0085        | [Convert f090 Mining Reserve Actor to a Keyless Account Actor](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0085.md)                                              | Accepted            |
| FIP0090        | [Non-Interactive PoRep](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0090.md)                                              | Accepted            | 

---

This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

**Comment by anorth** [comment]
Aren't some of those accepted FIPs now final in nv22?

**Comment by luckyparadise** [comment]
Hey @anorth ! I think what you are referencing is an outdated version. The correct FIPs and their statuses are correctly captured above.

**Comment by jaychoucc** [comment]
FIP0078
